lightshow/postprocess/parse.py

```diff
@@ -0,0 +1,361 @@
+import numpy as np
+import os
+from pathlib import Path
+
+def extract_FEFF(path): 
+    """Extract spectrum from FEFF output directory. 
+    
+    Parameters
+    ----------
+    path : str
+        FEFF output directory. 
+
+    Returns
+    -------
+    dict_output
+        A dictionary containing the spectrum. 
+        ``{"spectrum": [..., ...], "efermi": ..., "normalization_constant": ...}``
+
+    Raises
+    ------
+    OSError
+        If the output path does not contain ``xmu.dat`` file. 
+    """
+    
+    dict_output = {}    
+    path = Path(path)
+    
+    # Extract sepctrum 
+    try: 
+        spectrum = np.loadtxt(path / "xmu.dat")
+    except OSError as err: 
+        print(err)
+        return
+    dict_output['spectrum'] = np.column_stack((spectrum[:,0],spectrum[:,3]))
+
+    # Extract Fermi energy at k=0
+    efermi = spectrum[spectrum[:,2]==0,1]
+    try: 
+        dict_output['efermi'] = efermi[0]
+    except: 
+        print("Failed to extract Fermi energy.")
```

Comment:
What is the point of this try/except statement? Again I feel this is an error to let the user handle.

Reply:
Resolved in the updated code on 12.3.2023